streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/GraphNode.java

```diff
@@ -119,6 +120,11 @@ public Integer buildPriority() {
 
     public abstract void writeToTopology(final InternalTopologyBuilder topologyBuilder);
```

Comment:
I don't know if removing this method and replace it with the below method with the properties parameter is better. There are several classes that implement `GraphNode` and that will need to accept the new parameter. It is an internal interface and impl, though. Do you think I should go with that approach instead?

Reply:
Yeah. Might be worth to just remove this one even if we need to update all implementations.